src/Microsoft.ML.StandardLearners/Standard/LinearPredictor.cs

```diff
@@ -353,9 +353,13 @@ public void SaveAsCode(TextWriter writer, RoleMappedSchema schema)
             LinearPredictorUtils.SaveAsCode(writer, in weights, Bias, schema);
         }
 
-        public abstract void SaveSummary(TextWriter writer, RoleMappedSchema schema);
+        [BestFriend]
+        private protected abstract void SaveSummary(TextWriter writer, RoleMappedSchema schema);
```

Comment:
> private protected [](start = 7, length = 18)

You might need to keep this to internal. Does the  [Best Friend] extend to private protected, and not just internal? 

OLS using this, is in another assembly.